plugins/circleci-heroku/readme.md

```diff
@@ -0,0 +1,43 @@
+# `circleci-heroku` Tool Kit plugin
+
+WIP
+
+## Installation
+
+Install `@dotcom-tool-kit/circleci-heroku` as a `devDependency` in your app:
+
+```sh
+npm install --save-dev @dotcom-tool-kit/circleci-heroku
+```
+
+Add the plugin to your [Tool Kit configuration](https://github.com/financial-times/dotcom-tool-kit/blob/main/readme.md#configuration):
+
+```yaml
+plugins:
+	- '@dotcom-tool-kit/circleci-heroku'
+```
+
+### For Tool Kit generated Circle `config.yml`
+
+If you are migrating your project to Tool Kit for the first time then this plugin can generate a new `.circleci/config.yml` file for your project including Tool Kit configured workflows. To use this feature please delete or rename your existing Circle `config.yml` file before running the install command.
+
+Install this plugin's hooks:
```

Comment:
I have added that point specifically because it tripped me up. I _wanted_ to get a Tool Kit automatically generated `config.yml` file but one wasn't generated for me when I ran  `npx dotcom-tool-kit --install`. The reason was because I hadn't deleted my existing `config.yml` file first. This note is to tell users that that's what they need to do. Alternatively (or for a later enhancement) we could delete or rename their existing `config.yml` for them as part of the install command.

Reply:
Did the install command give a warning to say "we didn't generate a CircleCI config cause there is non tool kit one already"? Maybe that is another thing we could do along side this paragraph in this readme.

Reply:
No it didn't - that's a good idea. I'll create a card for it (might do it in innovation day!)